lib/requester/request-wrapper.js

```diff
@@ -147,20 +94,21 @@ var _ = require('lodash'),
      * @param cb
      */
     setCertificate = function (options, cb) {
```

Comment:
separate request options and additional params (from where we mutate request options) sent by us